frontend/build-configs/nuernberg/index.ts

```diff
@@ -32,6 +32,7 @@ export const nuernbergCommon: CommonBuildConfigType = {
         local: "http://localhost:8000",
     },
     appLocales: ['de', 'en'],
+    localeOverwritePath: 'assets/l10n/nuernberg',
```

Comment:
we have project specific assets in `assets/<project-name>` i think we should either keep that structure or also restructure the image assets

Reply:
Will fix in #1189, I just realized I also had typos in the file names (overwrite instead of override).